core/src/util/usSharedLibrary.cpp

```diff
@@ -111,6 +110,14 @@ SharedLibrary& SharedLibrary::operator =(const SharedLibrary& other)
   return *this;
 }
 
+bool SharedLibrary::IsSharedLibrary()
+{ // Testing for file extension isn't the most robust way to test
+  // for file type. Furthermore, the existence of this function
+  // is out of place; why test for a shared library if this class
+  // is supposed to represent a shared library?
```

Comment:
@saschazelzer, I agree. In the meantime, I am moving this function out of usSharedLibrary.cpp (because it doesn't make much logical sense) and into usUtils.cpp.